modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -1552,6 +1552,10 @@ export const console: ConsoleNS = {
                         }
                     },
                     certificateSection: {
+                        heading: "Certificates",
+                        description: "The certificate information provided here is used to verify signed responses from the external Identity Provider (IdP).",
+                        info: "This validation is applicable only to federated IdP-initiated logout and federated authentication flows " +
+                        "that use native SDK-based app-native authentication.",
```

Comment:
Just asking to clarify, which of the following is meant to be communicated from this info message?

- fidp init logouts in app native Auth flows and fidp Auth flows in app native Auth flows

Or 

- fidp init logouts in any Auth flows and fidp Auth flows in app native Auth flows

Reply:
the second one

Reply:
The way its currently written conveys the first one, IMO. To improve wording, we may add a comma as follows.

`This validation is applicable only to federated IdP-initiated logout, and federated authentication flows that use native SDK-based app-native authentication.`

On the other hand, we may be able to improve wording as follows.

`This validation is applicable only to federated Identity provider-initiated logout, and federated authentication flows that use app-native authentication.`

WDYT?

cc: @himeshsiriwardana

Reply:
+1 to the first one. we need to specifically mention the sdk based flow since the validation only happens in native mode as opposed to redirect mode. Also better to keep the length short IMO since it makes the UI look nicer